nodewatch/scripts/ci/lint.sh

```diff
@@ -0,0 +1 @@
+../../../linters/python/lint.sh
```

Comment:
Should add ``#!/bin/bash``

Reply:
it is actually symbolic link, so should be fine

Reply:
This is a symlink for the linter file.
The linter file itself already has `#!/bin/bash`